### ISSUE TITLE
dcat export requires triple quotes """ on strings that contain a newline

### ISSUE THREAD

**Issue by marc-portier** [gbif/ipt#1482]
See https://www.w3.org/TR/turtle/#h4_turtle-literals

String literals in ttl need triple quotes surounding them if they are to contain CR or LF  (i.e. newlines)

The current implementation does not check for that nor guarantee the proper escaping. As a consequence the generated format can not (always) be parsed.

Example showing the ttl (node bease turtle validator) `npm install -g turtle-validator`
```
$ curl --url https://ipt.inbo.be/dcat -o - 2>/dev/null |cat -n - | grep -P '^[^"]*"[^"]*$' |head -1
    84	dct:description "Ecological typology of waterways, plant surveys.

$ curl --url https://ipt.inbo.be/dcat -o - 2>/dev/null |ttl
ERROR: Unexpected ""Ecological" on line 84.
Validator finished with 0 warnings and 1 errors.
```

This affects the source code of the following files
```
./src/test/java/org/gbif/ipt/task/GenerateDCATTest.java
./src/main/java/org/gbif/ipt/task/GenerateDCAT.java
```

To resolve the issue:
* the ttl serialization should check for strings containing newlines, and properly wrap those with triple quotes (alternatively the checking could be skipped, and triple quotes used out of precaution) 
* the test should force at least one string with a newline to check for the proper escaping

Note that the code already seems to show some 'newline sensitivity' already --> https://github.com/gbif/ipt/blob/master/src/main/java/org/gbif/ipt/task/GenerateDCAT.java#L531
This however is only checking on the description-field that is expected to be multi-line, but still misses the fact that each of these 'lines' by themselves could have newlines inside.

On a more general note: one could consider using an RDF library as Apache Clerezza (http://clerezza.apache.org/getting-started/tutorial/tutorial-03/) or Apache Jena (https://jena.apache.org/documentation/io/rdf-output.html#jena_model_write_formats) to manage the proper RDF serialization. 

Such library approach also supports:
* serialization into other rdf formats (e.g. json-ld?) - something that could be offered through content-negotiation? 
* other escaping issues (like not containing " or """ )

**Comment by marc-portier** [comment]
Sorry for the explicit mention, @simon-vc @Amenai @MattBlissett,
but is there any likeliness that any further remark on this topic
(let alone me getting motivated to prep up some pull request) 
is eventually going to lead up to some actual fix / release?

What is the overall state of and plan for this project?

**Comment by mike-podolskiy90** [comment]
Thank you @marc-portier for reporting this issue. This will be included in the upcoming 2.5.0 version.
Closing this
